.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
#### Nitpick
Always add a new line after the last line

Reply:
this has been effected in all files. Thanks

Reply:
it has been effected.